lighthouse-core/audits/server-response-time.js

```diff
@@ -11,27 +11,27 @@ const MainResource = require('../computed/main-resource.js');
 
 const UIStrings = {
   /** Title of a diagnostic audit that provides detail on how long it took from starting a request to when the server started responding. This descriptive title is shown to users when the amount is acceptable and no user action is required. */
-  title: 'Server response times are low (TTFB)',
+  title: 'Server response times are low',
```

Comment:
```suggestion
  title: 'Server response times are short',
```

or 'quick'?

weird to say " a low amount of time'

Reply:
WDYT about fixing the plural wrinkle here too?